learning_assistant/api.py

```diff
@@ -187,3 +187,21 @@ def get_course_id(course_run_id):
     course_data = get_cache_course_run_data(course_run_id, ['course'])
     course_key = course_data['course']
     return course_key
+
+
+def get_message_history(course_id, user, message_count):
+    """
+    Given a course run id (str), user (User), and message count (int), return the associated message history.
+
+    Returns a number of messages equal to the message_count value.
+    """
+    # If the received message count exceeds the number of messages present, we return all the messages queried.
+    actual_message_count = LearningAssistantMessage.objects.count()
```

Comment:
I actually don't know if this is strictly necessary. Have you tested the code without this, and just indexing using the `message_count` value? Only asking because the same syntax is used here: https://github.com/openedx/edx-platform/blob/b20498cc4eec4ed1b85508967bef6c915372dd5b/openedx/core/djangoapps/notifications/tasks.py#L105

Reply:
Oh that actually works. I'll get rid of this code.